dotnet/Gherkin/CucumberMessages/Types/SourceReference.cs

```diff
@@ -5,10 +5,10 @@ namespace Gherkin.CucumberMessages.Types;
 public class SourceReference
 {
```

Comment:
A SourceReference should contain one of a Location, Uri, JavaMethod or a JavaStackTraceElement.

Reply:
thank you for feedback. I will make an optimal it then